plugins/config/lib/configService.js

```diff
@@ -1394,7 +1394,7 @@ function getFileListing(fileTable, startScope, endScope, relativePath, filename,
     try {
       var files = fs.readdirSync(path,{encoding:'utf8'});
       if (files) {
-        logger.debug("Directory files found="+files.length);
+        logger.debug("ZWED0234I", files.length); //logger.debug("Directory files found="+files.length);
```

Comment:
"ZWED0234I":"Directory files found=%", lacks a string, should be
  "ZWED0234I":"Directory files found=%s",

Reply:
Addressed.